### PR TITLE
chore(web): fix feedback card wording and font weight

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
Feedback card doesn't match the design
image

## Description
This PR updates the wording and styling of the in app satisfaction survey to better match the design.

We should try make this a bit more re-usable next time we add a new feature feedback survey.

### CODE REVIEW

**Review by Alportan — APPROVED** [review]
LGTM 🎉 Thanks for picking this up so fast Tony! 🫶

File: web/src/locales/en.json [hunk 1]
```diff
@@ -124,14 +124,16 @@
       "primary-question": "The description of the data estimation was easy to understand:"
     },
 
-    "primary-question": "How satisfied are you with your experience using the Electricity Maps app?",
+    "primary-question": "How would you rate your overall satisfaction with the Electricity Maps app?",
```
Comment (VIKTORVAV99) [inline web/src/locales/en.json:127]: Is this added to any other language that we need to adjust as well?

Comment (tonypls) [reply web/src/locales/en.json:127]: Yes it's also in 3 other languages which translate to "How satisfied are you with your experience using the Electricity Maps app?" I think it's a finer detail though, here's the Swedish: "Hur nöjd är du med din upplevelse av att använda Electricity Maps-appen?" Does it still capture the general question?

Comment (VIKTORVAV99) [reply web/src/locales/en.json:127]: Yeah I think that would still work. I can add the Swedish translations for the new strings if you wish?

Comment (tonypls) [reply web/src/locales/en.json:127]: yeah sounds good!

**Pull request merged**